pages/activities.jsx

```diff
@@ -46,13 +49,17 @@ var ActivityKit = React.createClass({
     // Displays the last author with an "and" and an oxford comma.
     var _this = this;
     var hasDash = this.props.developedBy.indexOf("-") > -1;
-    var developedByArray = this.props.developedBy.replace("-", ",").split(",");
+    var hasInPartnershipWith = this.props.developedBy.indexOf("in partnership with") > -1;
+    var developedByArray = this.props.developedBy.replace("-", ",").replace("in partnership with", ",").split(",");
```

Comment:
hmmmmmmmmmmm this will get interesting when we need to localize things... :smiling_imp: I think it's ok for now tho.